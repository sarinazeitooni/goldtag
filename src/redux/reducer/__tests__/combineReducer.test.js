import React from 'react';
import {shallow} from 'enzyme';
import rootReducer from "../combineReducer";
describe("rootReducer component", () => {
    const container = shallow(<rootReducer/>)
    it("it should render", () => {
        expect(container).toMatchSnapshot();
    });
});