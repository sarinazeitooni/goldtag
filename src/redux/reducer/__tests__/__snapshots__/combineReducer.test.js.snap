// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rootReducer component it should render 1`] = `ShallowWrapper {}`;
